pkgs/servers/sql/mariadb/connector-c/default.nix

```diff
@@ -14,7 +14,7 @@ in stdenv.mkDerivation {
   inherit version;
 
   src = fetchurl {
-    url = "https://downloads.mariadb.com/Connectors/c/connector-c-${version}/mariadb-connector-c-${version}-src.tar.gz";
+    url = "https://github.com/mariadb-corporation/mariadb-connector-c/archive/refs/tags/v${version}.tar.gz";
```

Comment:
Can we use `fetchFromGitHub`? because I think it is preferable it on dynamically generated archives.

Reply:
Sounds good! Switched to `fetchFromGitHub`.